push-athena-data-to-rds/athena/queries.go

```diff
@@ -83,27 +95,37 @@ func AssetypeDataQuery(queryParams map[string]string, db string, table string) (
 	if reqErr != nil {
 		return "", requestErrMsg
 	}
+
 	publisherDataStringQuery, publisherDataErrMsg := generateStringQuery(publisherDataSubQuery)
 	pubDataErr := publisherDataErrMsg.Err
 
 	if pubDataErr != nil {
 		return "", publisherDataErrMsg
 	}
 
+	refererStringQuery, refererErrMsg := generateStringQuery(refererSubQuery)
+	refererErr := refererErrMsg.Err
+
+	if refererErr != nil {
+		return "", refererErrMsg
+	}
+
 	countExp := sq.Expr("count(*)")
 	hitSumExp := sq.Expr("sum(case WHEN cache_status = 'hit' THEN 1 ELSE 0 end)")
 	responseByteSumExp := sq.Expr("sum(response_byte)")
 
 	query := sq.Select().
 		Prefix(requestStringQuery).
 		Prefix(publisherDataStringQuery).
+		Prefix(refererStringQuery).
 		Column("name AS publisher_name").
 		Column(sq.Alias(countExp, "total_requests")).
 		Column(sq.Alias(responseByteSumExp, "total_bytes")).
 		Column(sq.Alias(hitSumExp, "hit_count")).
+		Column("referer").
 		Column(dateQuery).
-		From("publisher_data").
-		GroupBy("name")
+		From("referer_name").
```

Comment:
Any other name for sub-table instead of `referer_name`?

Reply:
changed to `populate_referer`